package.json

```diff
@@ -81,28 +75,15 @@
     "scripts": {
         "build": "tsc",
         "build:watch": "tsc -w",
-        "test": "jest test --silent --maxWorkers=50%",
-        "test-with-coverage": "jest test --silent --maxWorkers=50% --coverage",
-        "test-watch": "jest test --silent --maxWorkers=25% --watch",
+        "test": "vitest run --config ./test/vitest.config.mts",
+        "test:coverage": "vitest run --config ./test/vitest.config.mts --coverage",
+        "test:watch": "vitest watch --config ./test/vitest.config.mts",
         "eslint": "eslint --max-warnings=0",
         "pretty:write": "prettier --write .",
         "pretty:check": "prettier --check .",
         "typedoc": "typedoc --gitRevision $(git describe --tag --abbrev=0) --tsconfig tsconfig.json --excludePrivate --excludeProtected --excludeExternals --entryPointStrategy expand ./src --sourceLinkTemplate 'https://github.com/Koenkk/zigbee-herdsman/blob/{gitRevision}/{path}#L{line}' -out typedoc",
         "clean": "rimraf temp coverage dist",
         "prepack": "pnpm run clean && pnpm run build"
     },
-    "version": "3.0.4",
-    "jest": {
-        "coverageThreshold": {
```

Comment:
How is this now defined?

Reply:
In https://github.com/Nerivec/zigbee-herdsman/blob/vitest/test/vitest.config.mts (I used the shorthand that puts it all at 100 for coverage.)